rasp/base.py

```diff
@@ -37,14 +37,37 @@ def get_page_source(self, url, data=None):
 
 class Webpage(object):
     def __init__(self, url=None, source=None):
-        self.url = url
-        self.source = source
+        """The Webpage object represents all we know about a single scraped page.
+
+        The Webpage object is the key object constructed by an engine to represent what we know about a given webpage.
+        It includes things like the page source, url, and date of access.
+
+        :param url: the url of the webpage you are representing
+        :param source: the source, as text of the webpage.
+        :return: Webpage
+        """
+
+        self._url = url
+        self._source = source
+        self._access_date = time.time()
+
+    @property
```

Comment:
Do we need to be using properties if there isn't any custom behavior to getting/setting these attributes?  Isn't that just extra code for no reason?